Cargo.toml

```
@@ -45,5 +45,15 @@ required-features = ["groth16"]
 name = "batch"
 harness = false
 
+[[bench]]
+name = "slow"
+harness = false
+
 [badges]
 maintenance = { status = "actively-developed" }
+
+[patch.crates-io]
+bls12_381 = { git = "https://github.com/nuttycom/bls12_381.git", branch = "bitvec-0.22" }
```

Once https://github.com/zkcrypto/bls12_381/pull/62 is merged:
```suggestion
bls12_381 = { git = "https://github.com/zkcrypto/bls12_381.git", branch = "main" }
```